src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -185,18 +186,27 @@
     @CheckForNull
     private String credentialsId;
 
+    @CheckForNull
+    private String credentialsIdRaw;
+
     /**
      * The repository owner.
      */
-    @NonNull
+    @CheckForNull
     private final String repoOwner;
 
     /**
      * The repository
      */
-    @NonNull
+    @CheckForNull
     private final String repository;
 
+    /**
+     * Raw URL for GitHub Server
+     */
+    @CheckForNull
+    final String rawUrl;
```

Comment:
Did you check to make sure there are no issues when deserializing old instances of this class after upgrading the plugin to a version that includes this PR?

Reply:
Yes, there are some previous tests already doing that, with no rawUrl field in the xml file.